cocotb/share/lib/fli/FliObjHdl.cpp

```diff
@@ -249,14 +249,41 @@ int FliEnumObjHdl::set_signal_value(const int32_t value,
     }
 
     if (m_is_var) {
-        mti_SetVarValue(get_handle<mtiVariableIdT>(),
-                        static_cast<mtiLongT>(value));
+        switch (action) {
+            case GPI_DEPOSIT:
+                mti_SetVarValue(get_handle<mtiVariableIdT>(),
+                                static_cast<mtiLongT>(value));
+                return 0;
+            case GPI_FORCE:
+                LOG_ERROR("Cannot force a VHDL variable");
```

Comment:
Maybe reword these messages to
```suggestion
                LOG_ERROR("Forcing VHDL variables is not supported by FLI");
```
to prevent users reading this as a message about a specific variable (might happen if they are not native speakers).

Reply:
Perhaps instead:
```suggestion
                LOG_ERROR("Cannot force VHDL variables");
```

VHDL variables can only be set, they do not support inertial writes or force/release in the language. IDK how the VHPI handles these cases, perhaps instead we should treat force as a regular variable set so it's consistent?

Reply:
VHPI allows forcing variables. Section 22.5.2 of the LRM has details.

Reply:
So there is no language-level support for forcing variables, but you can do it with the VHPI? Okay. I guess the first suggestion is more correct.